packages/utilities/src/aria.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * ARIA helper to concatenate attributes, returning undefined if all attributes
+ * are undefined. (Empty strings are not a valid ARIA attribute value.)
+ *
+ * NOTE: This function will NOT insert whitespace between provided attributes.
```

Comment:
Because all of the uses present in the codebase were already inserting spaces. This is a helper primarily to return undefined when all values are undefined, as opposed to `'undefined'` string value or `''`.